fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/ControlTokens.kt

```diff
@@ -66,6 +70,8 @@ class ControlTokens {
                 ControlType.SearchBarPersonaChip -> SearchBarPersonaChipTokens()
                 ControlType.SearchBar -> SearchBarTokens()
                 ControlType.Shimmer -> ShimmerTokens()
+                ControlType.TabBar -> TabBarTokens()
+                ControlType.TabBarTabItem -> TabBarTabItemsTokens()
```

Comment:
Instead of making it a public control token. Should we add it as a customized token of TabItem in the control's kt file itself.